### DISCUSSION TITLE
How shall I directly open the download path from UI? 如何在UI里直接打开下载路径？

### DISCUSSION THREAD

**Discussion by kyleli666** [alexhua/Aria2-for-chrome#131] [Q&A] [answered]
I feel this may be an easy one, but just cannot figure it out 😭 
How shall I directly open this download path with Windows Explorer?
![image](https://user-images.githubusercontent.com/93997542/229527764-72edeb96-9853-4252-abc7-6b4b1fbf0844.png)
It looks like I can click on it, but nothing happens.

**Comment by alexhua** [comment, marked as answer]
Can't not open Windows Explorer from extension. Because extension has no capability to implement this.

**Reply by kyleli666** [reply]
Okay, thanks a lot~
